partcad/src/partcad/context.py

```
@@ -423,18 +424,20 @@ def get_all_packages(self, parent_name=None):
         self.import_all(parent_name)
         return self.get_packages(parent_name)
 
-    def get_packages(self, parent_name=None):
+    def get_packages(self, parent_name: str = None) -> list[dict[str, str]]:
```

Ruff (0.8.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)